### PR TITLE
Gpio ctl

### PR DESCRIPTION
**Pull request by modalai-tom**
### Solved Problem
When ... I found that ...

Fixes #{Github issue ID}

### Solution
- Add ... for ...
- Refactor ...

### Changelog Entry
For release notes:
```
Feature/Bugfix XYZ
New parameter: XYZ_Z
Documentation: Need to clarify page ... / done, read docs.px4.io/...
```

### Alternatives
We could also ...

### Test coverage
- Unit/integration test: ...
- Simulation/hardware testing logs: https://review.px4.io/

### Context
Related links, screenshot before/after, video

### CODE REVIEW

File: src/drivers/actuators/voxl_esc/voxl_esc.hpp [hunk 1]
```diff
@@ -133,6 +133,17 @@ class VoxlEsc : public ModuleBase<VoxlEsc>, public OutputModuleInterface
 
 	static constexpr uint16_t VOXL_ESC_NUM_INIT_RETRIES = 3;
 
+	static constexpr float    VOXL_ESC_GPIO_CTL_DISABLED_SETPOINT = -0.1f;
+	static constexpr float    VOXL_ESC_GPIO_CTL_THRESHOLD = 0.0f;
+
+	static constexpr uint32_t VOXL_ESC_GPIO_CTL = 0;
```
Comment (katzfey) [inline src/drivers/actuators/voxl_esc/voxl_esc.hpp:139]: Doesn't look like this one is ever used.

File: src/drivers/actuators/voxl_esc/voxl_esc.cpp [hunk 2]
```diff
@@ -1240,11 +1254,56 @@ bool VoxlEsc::updateOutputs(bool stop_motors, uint16_t outputs[MAX_ACTUATORS],
...
 			_extended_rpm);
 
+
 	if (_uart_port->uart_write(cmd.buf, cmd.len) != cmd.len) {
 		PX4_ERR("VOXL_ESC: Failed to send packet");
 		return false;
 	}
 
+	// Track and manage gpio command writes
+    	bool write_gpio_command = false;
```
Comment (katzfey) [inline src/drivers/actuators/voxl_esc/voxl_esc.cpp:1264]: In the source file this line has some spaces and a tab before it and the alignment is off

File: src/drivers/actuators/voxl_esc/voxl_esc.hpp (hunk 1 above)
Comment (modalai-tom) [reply src/drivers/actuators/voxl_esc/voxl_esc.hpp]: removed

File: src/drivers/actuators/voxl_esc/voxl_esc.cpp (hunk 2 above)
Comment (modalai-tom) [reply src/drivers/actuators/voxl_esc/voxl_esc.cpp]: fixed

**Pull request merged**